### PR TITLE
post-cfp

### PR DESCRIPTION
**Pull request by hypha**

### CODE REVIEW

**Comment by hypha** [comment]
Waiting for reviews. Do not merge.

File: data/faq.md [hunk 1]
```diff
@@ -31,40 +53,40 @@ A. To get an idea of what to expect, please have a look at the
 [EP2021 programme](https://ep2021.europython.eu/events/sessions/) and the
 [EP2019 programme](https://ep2019.europython.eu/events/sessions/).
 
-To select talks, we run a Call for Proposals (CFP). Talks are then curated based
-on the input from the community voting and expert panel reviews. Make sure to subscribe to our
-[EuroPython Blog](https://blog.europython.eu/#/portal/signup) for updates.
+The conference will feature around 20 tutorials/workshops, 120 talk sessions and other interactive sessions such as Posters, Helpdesks, Panel Discussions and Maker Space.
```
Comment (ntoll) [inline data/faq.md:56]: "and a Maker Space" (there's only one maker space, not several?)

File: data/faq.md [hunk 2]
```diff
@@ -31,40 +53,40 @@ A. To get an idea of what to expect, please have a look at the
...
 
-We aim to have a very diverse set of talks, tutorials, workshops, posters and
-keynotes, covering many different angles of where and how Python can be used to
-solve problems or do interesting things.
+We aim to have a very diverse programme, covering many different angles of where and how Python can be used to solve problems or do interesting things.
```
Comment (ntoll) [inline data/faq.md:58]: "covering the many different aspects of where and how Python solves problems or engages in interesting things." ???

File: data/faq.md [hunk 3]
```diff
@@ -31,40 +53,40 @@ A. To get an idea of what to expect, please have a look at the
...
 
 ## **Q. When do you accept talk proposals?**
 
-A. The [Call for Proposals](https://program.europython.eu/europython-2022/cfp) (CFP) opened on 8 March.
+A. The [Call for Proposals](https://program.europython.eu/europython-2022/cfp) (CFP) ran between 8 March and 3 April.
```
Comment (ntoll) [inline data/faq.md:62]: "The Call for Proposals (CFP) has closed. It ran between the 8th March - 3rd April." (just making sure folks don't try to submit late.)

File: data/faq.md [hunk 4]
```diff
@@ -31,40 +53,40 @@ A. To get an idea of what to expect, please have a look at the
...
+
+We are thrilled to have received a record high of 429 proposals on a diverse range of topics.
+
+## **Q. How do you review and select talks?**
+
+A. We believe that a good conference consists of popular talks as well as new or niche topics - all by and for our diverse community. Thus after Call for Proposals (CFP) finishes, submissions go through two round of unbiased reviews: speakers' names will not be displayed.
```
Comment (ntoll) [inline data/faq.md:68]: "Thus, "

Hmm... reviews are always biased towards the reviewers point of view. ;-) How about "go through two rounds of review and refinement as we put together our programme. This is a "blind" process, so speakers' names will not be displayed to the reviewers." ..? I use "refinement" in the sense that the programme AND the talks are being refined as stuff gets cut, included or changed. Does that make sense..?

File: data/faq.md [hunk 5]
```diff
@@ -31,40 +53,40 @@ A. To get an idea of what to expect, please have a look at the
...
+
+- community voting: open to all 2015-2022 EuroPython conference ticket holders + 2022 talk proposal submitters. Submissions are in random order for each voter. Please check out our [talk voting page](https://ep2022.europython.eu/voting) for details.
+
+- panel feedback: we invite a diverse group of experienced reviewers to refine & improve on submissions within their areas of expertise. The feedback helps the programme team make informed decisions while evaluating the quality of each submission. [Submit your interest here](https://t.co/JyOdGb71f8)
```
Comment (ntoll) [inline data/faq.md:72]: No need for "we invite". Why not: "a diverse groups of experienced reviewers will refine and give feedback on the submissions within their realm of expertise." There's no guarantee they'll improve the submissions..!

"Their insights help the programme team refine and plan a programme that truly reflects, informs and stimulates our EuroPython family."

File: data/faq.md [hunk 6]
```diff
@@ -31,40 +53,40 @@ A. To get an idea of what to expect, please have a look at the
...
 
-It has been extended for another week. New deadline: **3 April AoE**.
+The programme team will curate the final programme based on input from community voting, panel feedback and other editorial criteria such as the diversity of topics and speakers.
```
Comment (ntoll) [inline data/faq.md:74]: "Ultimately, the programme team curate the final programme based upon these important factors: community voting, panel feedback and editorial criteria such as the diversity, depth and breadth of topics and speakers."

(Shorter but says the same?)

File: data/faq.md [hunk 7]
```diff
@@ -31,40 +53,40 @@ A. To get an idea of what to expect, please have a look at the
...
-We are thrilled
-to receive proposals on any topic that could be of interest to our community.
+[Subscribe to receive email updates](https://blog.europython.eu/#/portal/signup)
 
-Especially if you are considering becoming a speaker for the first time, or if
-you are from an underrepresented group, we strongly encourage you to send in
-your proposals. We ran a
-[Speaker Mentorship Programme](https://ep2022.europython.eu/mentorship) and matched 15 mentees and mentors. Check out the [AMA session](https://www.youtube.com/watch?v=18donGcl0Jg) as part of the Mentorship Programme. Email us at ([program@europython.eu](mailto:program@europython.eu)) to tell us what other broader support you might need!
+## **Q. What support do you give to speakers?**
+A. If accepted, speakers are eligible for a free conference ticket and trainers a free combined ticket. In addition, speakers are encouraged to apply for additional [Financial Aid](https://ep2022.europython.eu/finaid), should there be a need.
```
Comment (ntoll) [inline data/faq.md:79]: I'd split the sentence at the start (just to clarify the difference):

"If accepted, speakers are eligible for a free conference ticket. Workshop organisers are eligible for a free combined ticket."

File: data/faq.md [hunk 8]
```diff
@@ -31,40 +53,40 @@ A. To get an idea of what to expect, please have a look at the
...
 
-If you have something in mind, there is no better time to start preparing your
-proposal! For more insights, please have a look at the
-[EP2019 CFP page](https://ep2019.europython.eu/events/call-for-proposals/),
-which was our last in-person conference. The overall structure for EP2022 remains the same.
+We want to give extra support to first-time speakers, especially the ones from an underrepresented group. It is in this spirit that we ran a
```
Comment (ntoll) [inline data/faq.md:81]: We give support to first-time speakers and actively encourage, accommodate and welcome the participation of those from under-represented groups in tech. For example, our Speaker Mentorship Programme matched 15 mentees with mentors. Check out... etc

File: data/faq.md [hunk 9]
```diff
@@ -93,18 +118,22 @@ A. Yes, our
...
 
 ## **Q. When can sponsors start signing up?**
 
-A. We are aiming to start sponsor signup in late March. Please check
-[2019 Sponsor Packages](https://ep2019.europython.eu/sponsor/packages/) to get an idea of the sponsorship options. The packages are not finalised yet, but will be similar to what we had in 2019. Sponsors who sign up early will again receive a 10% discount.
+A. Sponsors are welcome to sign up now! You can find all the details on our [sponsor page](https://ep2022.europython.eu/sponsor). Sponsors who sign up by 29 April will again receive a 10% Early Bird discount.
 
-We are committed to making EuroPython sponsorship accessible to most organisations keen to help the community.
+We are committed to making EuroPython sponsorship accessible to most organisations keen to help the community. [SME discount](https://ep2022.europython.eu/sponsor#sme) and [Free Table for Open Source Projects](https://ep2022.europython.eu/sponsor#free-table-for-open-source-projects) are offerings driven by our community spirit.
```
Comment (ntoll) [inline data/faq.md:134]: "accessible to as many organisations as possible, who are keen to help the community."

**Review by ntoll — APPROVED** [review]
@hypha - this is wonderful :tada:

As always, my annotations are suggestions and please take anything I say with a pinch of salt. Where necessary, I've tried to explain my change if it isn't clear what I'm trying to achieve (usually, more clarity in less words, simply put).

Does this help?

File: data/faq.md (hunk 1 above)
Comment (hypha) [reply data/faq.md:56]: touché!

File: data/faq.md (hunk 3 above)
Comment (hypha) [reply data/faq.md:62]: Great point!

File: data/faq.md (hunk 4 above)
Comment (hypha) [reply data/faq.md:68]: Thank you thank you! Makes total sense! I was thinking "unbiased" is a big word! :P

File: data/faq.md (hunk 6 above)
Comment (hypha) [reply data/faq.md:74]: Thank you for the refinement ;)

File: data/faq.md (hunk 7 above)
Comment (hypha) [reply data/faq.md:79]: Aye aye!

File: data/faq.md (hunk 8 above)
Comment (hypha) [reply data/faq.md:81]: This is so much better. Thank you!

File: data/faq.md (hunk 9 above)
Comment (hypha) [reply data/faq.md:134]: Big yes!

**Comment by hypha** [comment]
@patrick91 could you review if the [css changes](https://github.com/EuroPython/website/pull/36/files#diff-7421d78623e713232310ca42d646b242853cb9bab6e64e420fc8a94ce02ccd20) are OK? Those are for the tickmarks in [FAQ](https://github.com/EuroPython/website/pull/36/files#diff-fb3bc4b5e563548311070e266358d025bbd76c3f11742d79f19472207e5ad034).

Also made changed absolute links to relative ones in the .md files for better reusability next years. Could you confirm this is not problematic?

**Comment by patrick91** [comment]
@hypha looks great!

**Review by patrick91 — APPROVED** [review]

**Comment by patrick91** [comment]
feel free to merge it when you're ready :)

**Pull request merged**